api-samples/privacy/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Privacy API sample",
+  "version": "1.0",
+  "manifest_version": 3,
+  "background": {
+    "service_worker": "service-worker.js"
+  },
+  "permissions": ["privacy"],
+  "action": {}
```

Comment:
Is this needed?

Reply:
Ah got, we're programmatically changing the action which is why we don't need more here.